docs/source/security/index.rst

```diff
@@ -0,0 +1,103 @@
+Security Integration
+====================
+MapStore is integrated with the geOrchestra security infrastructure. This happens thanks to:
+ * an authentication filter using the geOrchestra security proxy headers to authenticate the user and assign proper MapStore groups and roles
+ * LDAP enabled DAOs to get available roles from the geOrchestra LDAP repository
+
+Authentication Filter
+^^^^^^^^^^^^^^^^^^^^^
+The authentication filter intercepts every MapStore backend request to extract the headers forwarded by the geOrchestra security-proxy,
+and use them to properly authenticate and authorize the current user, in particular:
+ * sec-username is used to authenticate the current user (anonymous access is assigned if the header does not exist)
+ * sec-roles is used to assign MapStore groups to the current user (groups will be used by the admin to assign permissions for the MapStore resources, e.g. maps)
+ * a particular role (MAPSTORE_ADMIN) is mapped to the MapStore ADMIN role
```

Comment:
Here I have this warning: 
```
mapstore2-georchestra/docs/source/security/index.rst:11: WARNING: Unexpected indentation.
```
Stange because the other lines are ok. I added a empty line between the top and remove the space at the start (all list, and it worked, like this

```rst
The authentication filter intercepts every MapStore backend request to extract the headers forwarded by the geOrchestra security-proxy,
and use them to properly authenticate and authorize the current user, in particular:

 * sec-username is used to authenticate the current user (anonymous access is assigned if the header does not exist)
 * sec-roles is used to assign MapStore groups to the current user (groups will be used by the admin to assign permissions for the MapStore resources, e.g. maps)
 * a particular role (MAPSTORE_ADMIN) is mapped to the MapStore ADMIN role
```

Reply:
fixed